redisinsight/ui/src/pages/browser/BrowserPage.tsx

```diff
@@ -304,88 +305,52 @@ const BrowserPage = () => {
           handleCreateIndexPanel={handleCreateIndexPanel}
         />
       </div>
-      <div
-        className={cx(styles.main, {
-          [styles.mainWithBackBtn]:
-            arePanelsCollapsed && isRightPanelOpen && !isBrowserFullScreen,
-        })}
-      >
-        <div className={styles.resizableContainer}>
-          <EuiResizableContainer
-            onPanelWidthChange={onPanelWidthChange}
-            style={{ height: '100%' }}
+      <div className={cx(styles.main)}>
+        <PanelGroup className={styles.panelGroup} direction="horizontal" onLayout={onPanelWidthChange}>
+          <Panel
```

Comment:
I see this is a draft PR, but yet, I wanted to point this out. So, IMO, let's have a custom component that exports `PanelGroup` and `Panel`, and `PanelResizeHandle` in `uiSrc/components/...`. This way we gain:
- common styling in a single place
- if, for some reason, we decide to switch `react-resizable-panels` with something else, it's going to be in a single place

Additionally, we can customise it as per our needs - we know that we have 2 panels and one of them + the handle can be "invisible", so that logic can live there.